Casks/i/impel.rb

```diff
@@ -0,0 +1,32 @@
+cask "impel" do
+  version "1.0.3"
+  sha256 "627e6e5e4699fab4130fc7a59fbcd6d223317bc6f6e9fe16ad0fe20ab60b690d"
+
+  url "https://impel-sparkle-updater.fly.dev/impel_#{version.dots_to_underscores}.tar.xz", verified: "impel-sparkle-updater.fly.dev/"
```

Comment:
The `verified` should be on its own line, indented to the quote.  Thanks for updating!
```suggestion
  url "https://impel-sparkle-updater.fly.dev/impel_#{version.dots_to_underscores}.tar.xz",
      verified: "impel-sparkle-updater.fly.dev/"
```